File: GabrielScarpelinDiniz/27/removeElement.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        i = 0
        length = len(nums)
        while i < length:
            if nums[i] == val:
                nums.pop(i)
                i -= 1
                length = len(nums)
            i += 1
        return len(nums)

File: Leonardo Ogata/27/removeElement.java
import java.util.ArrayList;

class Solution {
    public int removeElement(int[] nums, int val) {
        ArrayList<Integer> filteredList = new ArrayList<>();

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                filteredList.add(nums[i]);
            }
        }

        for (int i = 0; i < filteredList.size(); i++) {
            nums[i] = filteredList.get(i);
        }

        return filteredList.size();
    }
}